src/test/java/uk/gov/hmcts/reform/orgrolemapping/helper/TestDataBuilder.java

```diff
@@ -540,7 +557,6 @@ public static AppointmentV2 buildAppointmentWithParamsV2(String epimms, String i
                 .appointmentType(appointmentType)
                 .startDate(startDate)
                 .endDate(endDate)
-                //.roles(roles)
```

Comment:
does the roles assignment need an equivalent in V2 or not? Also looks like like this builder could do with a tidy up as there are a few commented lines

Reply:
This has been refactored in upstream PR #1530, see 53adc1d84ad44a13a4edfda8dc89e71976f97c75.